contracts/AssetPool.sol

```diff
@@ -578,16 +583,6 @@ contract AssetPool is Ownable, IAssetPool {
         uint256 tokensToMint = (amountToDeposit * covSupply) /
```

Comment:
now we can just: `return (amountToDeposit * covSupply) / collateralBalance`

Reply:
Right, I'll fix it.

Reply:
ada4c54ebda5ebefc65f5a9a9b26ef04440a5971 sir